src/pages/home/report/ReportActionCompose.js

```diff
@@ -135,6 +143,10 @@ class ReportActionCompose extends React.Component {
     }
 
     render() {
+        // We want to make sure to disable on small screens because in iOS safari the keyboard up/down buttons will
+        // focus this from the chat switcher. #1228
```

Comment:
Sorry, that was just github collapsing the link to the issue, but this should be the full URL.

```suggestion
        // focus this from the chat switcher. https://github.com/Expensify/Expensify.cash/issues/1228
```